packages/eas-cli/src/graphql/queries/AppQuery.ts

```diff
@@ -60,4 +73,114 @@ export const AppQuery = {
     assert(data.app, 'GraphQL: `app` not defined in server response');
     return data.app.byFullName;
   },
+  async byIdWorkflowsAsync(
+    graphqlClient: ExpoGraphqlClient,
+    appId: string
+  ): Promise<WorkflowFragment[]> {
+    const data = await withErrorHandlingAsync(
+      graphqlClient
+        .query<AppByIdWorkflowsQuery>(
+          gql`
+            query AppByIdWorkflowsQuery($appId: String!) {
+              app {
+                byId(appId: $appId) {
+                  id
+                  workflows {
+                    id
+                    ...WorkflowFragment
+                  }
+                }
+              }
+            }
+            ${print(WorkflowFragmentNode)}
+          `,
+          { appId },
+          { additionalTypenames: ['App'] }
+        )
+        .toPromise()
+    );
+    assert(data.app, 'GraphQL: `app` not defined in server response');
+    return data.app.byId.workflows;
+  },
+  async byIdWorkflowRunsAsync(
+    graphqlClient: ExpoGraphqlClient,
+    appId: string,
+    limit?: number
+  ): Promise<WorkflowRunFragment[]> {
+    validateLimit(limit);
+    const data = await withErrorHandlingAsync(
+      graphqlClient
+        .query<AppByIdWorkflowRunsQuery>(
+          gql`
+            query AppByIdWorkflowRunsQuery($appId: String!, $limit: Int!) {
+              app {
+                byId(appId: $appId) {
+                  id
+                  runs: workflowRunsPaginated(last: $limit) {
+                    edges {
+                      node {
+                        id
+                        ...WorkflowRunFragment
+                      }
+                    }
+                  }
+                }
+              }
+            }
+            ${print(WorkflowRunFragmentNode)}
+          `,
+          { appId, limit },
+          { additionalTypenames: ['App'] }
+        )
+        .toPromise()
+    );
+    assert(data.app, 'GraphQL: `app` not defined in server response');
+    return data.app.byId.runs.edges.map(edge => edge.node);
+  },
+  async byIdWorkflowRunsFilteredByStatusAsync(
+    graphqlClient: ExpoGraphqlClient,
+    appId: string,
+    status: WorkflowRunStatus,
+    limit?: number
+  ): Promise<WorkflowRunFragment[]> {
+    validateLimit(limit);
+    const data = await withErrorHandlingAsync(
+      graphqlClient
+        .query<AppByIdWorkflowRunsFilteredByStatusQuery>(
+          gql`
+            query AppByIdWorkflowRunsFilteredByStatusQuery(
+              $appId: String!
+              $status: WorkflowRunStatus!
```

Comment:
ditto, should be able to merge these queries (I think?):


```suggestion
    status?: WorkflowRunStatus,
    limit?: number
  ): Promise<WorkflowRunFragment[]> {
    validateLimit(limit);
    const data = await withErrorHandlingAsync(
      graphqlClient
        .query<AppByIdWorkflowRunsFilteredByStatusQuery>(
          gql`
            query AppByIdWorkflowRunsFilteredByStatusQuery(
              $appId: String!
              $status: WorkflowRunStatus
```